firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -41,7 +41,7 @@ class FirebaseSessions internal constructor(firebaseApp: FirebaseApp) {
 
   private fun initiateSessionStart() {
     val sessionState = sessionGenerator.generateNewSession()
-    val sessionEvent = SessionEvent.sessionStart(sessionState)
+    val sessionEvent = SessionEvents.sessionStart(sessionState)
 
     Log.i(TAG, "Initiate session start: $sessionEvent")
```

Comment:
Curious on Kotlin capability - Just printing `$sessionEvent` would parse through the whole structure? If so, this is just cool!

Reply:
Yup! It does the toString for us, including calling toString on all the nested fields.